Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/userflowlanguagepage.md

---
title: "userFlowLanguagePage resource type"

> description: "User flow language pages are used determine the strings users will be shown during the user journey you have configured using user flows."

Reviewer: ```suggestion
description: "Determines the strings that are shown to users during a user flow."
```

author: "jkdouglas"
localization_priority: Normal
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# userFlowLanguagePage resource type

Namespace: microsoft.graph

> User flow language pages are used determine the strings users will be shown during the user journey you have configured using user flows. These language pages include both the default language translations provided by Microsoft, or custom pages that can be created to customize the language translations.

Reviewer: ```suggestion
Determines the user flow language pages that are shown to users during a user flow. These language pages include both the default language translations provided by Microsoft, or custom pages that can be created to customize the language translations.
```
